### PR TITLE
Make Silver SortedSets better

### PR DESCRIPTION
**Pull request by Aadit-Ambadkar**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [ ] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 1]
```diff
@@ -261,9 +266,12 @@ In addition to all of the regular set operations,
...
   multiset. However, this method takes time **linear** in the number of matches
   so you shouldn't use it in a contest.
-- If you want to remove a value _once_, make sure to use
-  `multiset.erase(multiset.find(val))` rather than `multiset.erase(val)`. The
-  latter will remove _all_ instances of `val`.
+- To remove a value __once__, use `ms.erase(ms.find(val))`.
+- To remove __all__ values, use `ms.erase(val)`
+
+<Warning>
+	Using `ms.erase(val)` erases __all__ instances of `val` from the multiset. To remove one value, use `ms.erase(ms.find(val))`!
```
Comment (bqi343) [inline content/3_Silver/Intro_Sorted_Sets.mdx:273]:
```suggestion
	Using `ms.erase(val)` erases __all__ instances of `val` from the multiset. To remove one instance of `val`, use `ms.erase(ms.find(val))`!
```

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: intro-sorted-sets
 title: 'More Operations on Sorted Sets'
-author: Darren Yao, Benjamin Qi, Andrew Wang
+author: Darren Yao, Benjamin Qi, Andrew Wang, Aadit Ambadkar
```
Comment (bqi343) [inline content/3_Silver/Intro_Sorted_Sets.mdx:4]: I don't think this warrants adding yourself as an author. You can add yourself as a contributor though (like in this module: https://github.com/cpinitiative/usaco-guide/blob/f352643e443adea14c1e83eaa07eaf158b3bd87f/content/3_Silver/DFS.mdx)

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 3]
```diff
@@ -117,7 +122,7 @@ which can only be used once before calling the `next()` method.
 
 </LanguageSection>
 
-## Sorted Sets
+## Sorted Sets (and the additional functions that come with them)
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Sorted_Sets.mdx:125]: not needed

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 4]
```diff
@@ -434,29 +442,29 @@ void modify(int x) {
...
+		ret.erase(ret.find(x - a)), ret.erase(ret.find(b - x)); // update ret
+		ret.insert(b - a);
 		dif.erase(it); // remove x from dif
 	} else { // x is not currently in dif, insert it
 		it = dif.insert(x).first; // insert x into dif
 		// it = iterator corresponding to x
 		int a = *prev(it), b = *next(it);
-		ret.erase(ret.find(b-a)); // update ret
-		ret.insert(x-a), ret.insert(b-x);
+		ret.erase(ret.find(b - a)); // update ret
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Sorted_Sets.mdx:452]: two spaces after code before starting an inline comment

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 5]
```diff
@@ -434,29 +442,29 @@ void modify(int x) {
...
 int main() {
 	ios_base::sync_with_stdio(0); cin.tie(0);
 	cin >> s >> m;
 	dif.insert(0); dif.insert(sz(s));
-	for (int i = 0; i < sz(s)-1; ++i)
-		if (s[i] != s[i+1]) dif.insert(i+1); // initialize dif
-	for (auto it = dif.begin(); next(it) != dif.end(); it ++)
-		ret.insert(*next(it)-*it); // initialize ret
+	for (int i = 0; i < sz(s) - 1; ++i)
+		if (s[i] != s[i + 1]) dif.insert(i + 1); // initialize dif
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Sorted_Sets.mdx:462]: braces

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 6]
```diff
@@ -483,45 +491,44 @@ class BitInversion{
...
-	public static void modify(int x){
-		if(x == 0 || x == s.length()) return;
-		if(dif.contains(x)){ // x is currently present in dif, remove it
+	public static void modify(int x) {
+		if (x == 0 || x == s.length()) return;
+		if (dif.contains(x)) { // x is currently present in dif, remove it
 			int a = dif.lower(x), b = dif.higher(x);
-			remove(x-a); remove(b-x); // update ret
-			add(b-a);
+			remove(x-a); remove(b - x); // update ret
```
[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply content/3_Silver/Intro_Sorted_Sets.mdx:498]: i need one of those _right now_.

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 7]
```diff
@@ -580,13 +587,13 @@ int main() {
...
 	cin >> s >> m;
 	dif.insert(0); dif.insert(sz(s));
-	for (int i = 0; i < sz(s)-1; ++i)
-		if (s[i] != s[i+1]) dif.insert(i+1);
-	for (auto it = dif.begin(); next(it) != dif.end(); it ++)
-		ad(*next(it)-*it);
+	for (int i = 0; i < sz(s) - 1; ++i)
+		if (s[i] != s[i + 1]) dif.insert(i+1);
+	for (auto it = dif.begin(); next(it) != dif.end(); it++)
+		ad(*next(it) - *it);
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Sorted_Sets.mdx:593]: braces, so many braces
also is `ad` explained the name seems awfully obscure

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Review by ryanchou-dev — COMMENTED** [review]
address other comments and ill approve

File: content/3_Silver/Intro_Sorted_Sets.mdx [hunk 8]
```diff
@@ -483,45 +491,44 @@ class BitInversion{
 	public static int m;
 	public static TreeSet<Integer> dif = new TreeSet<Integer>();
```
Comment (nathangong) [inline content/3_Silver/Intro_Sorted_Sets.mdx:492]:
```suggestion
	public static Set<Integer> dif = new TreeSet<Integer>();
```

File: content/3_Silver/Intro_Sorted_Sets.mdx (hunk 2 above)
Comment (Aadit-Ambadkar) [reply content/3_Silver/Intro_Sorted_Sets.mdx:4]: Hm but I did make a lot of changes and additions to the page. If you really insist then I'll make it a contributor

Comment (bqi343) [reply content/3_Silver/Intro_Sorted_Sets.mdx:4]: doing what an autoformatter would have done doesn't count

Comment (ryanchou-dev) [reply content/3_Silver/Intro_Sorted_Sets.mdx:4]: yeah i agree ^^

listing yourself as a contributor would be more reasonable

**Review by ryanchou-dev — APPROVED** [review]
i don't actually use java, but content looks ok

File: content/3_Silver/Intro_Sorted_Sets.mdx (hunk 2 above)
Comment (Aadit-Ambadkar) [reply content/3_Silver/Intro_Sorted_Sets.mdx]: done!

**Review by bqi343 — APPROVED** [review]

**Pull request merged**